app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -354,6 +354,16 @@ class Settings(
         false
     )
 
+    var showSecretScreen = false
```

Comment:
We have an `AppStore` that lives the lifetime of the app, we could use that to hold state too.

This can help in the future because, if a feature needs to know if we have secret settings enabled we could do this with `components.appStore.state.secretSettingsEnabled`.

This is an example of a Fenix pattern that we shouldn't follow as it only exists because it hasn't been cleaned up yet.